spark/src/main/scala/ai/chronon/spark/Analyzer.scala

```diff
@@ -287,7 +287,16 @@ class Analyzer(tableUtils: TableUtils,
       (analysis, leftDf)
     }
 
-    if (!skipTimestampCheck) {
+    val gbTables = ListBuffer[String]()
+    joinConf.joinParts.toScala.foreach { part =>
+      gbTables ++= part.groupBy.sources.toScala.map(_.table)
+    }
+
+    val noAccessTables = if (validateTablePermission) {
+      runTablePermissionValidation((gbTables.toList ++ List(joinConf.left.table)).toSet)
+    } else Set()
+
+    if (!skipTimestampCheck && noAccessTables.isEmpty) {
```

Comment:
can we print a warning error if we are skipping the timestamp check because there are tables with permission issues?